### ISSUE TITLE
--ignore not working with wildcards after recent refactor

### ISSUE THREAD

**Issue by lukebakken** [vim-erlang/vim-erlang-tags#35]
To reproduce:

* Clone https://github.com/erlang/otp.git
* Run the following from the cloned dir:

```
vim_erlang_tags.erl --ignore '*/dialyzer/*'
```

Output:

```
File ""*/dialyzer/*"" is not a proper file.
```

Caused by this commit:

https://github.com/vim-erlang/vim-erlang-tags/commit/1fb0a9dc5a88c568cbd9a25417edfdbaa52a5a4a

https://github.com/vim-erlang/vim-erlang-tags/blob/master/bin/vim_erlang_tags.erl#L249-L258

The previous behavior allowed for wildcards, as indicated by the help doc:

```
  -i, --ignore FILE_WILDCARD
                Ignore the files/directories that match the given wildcard.
                Read http://www.erlang.org/doc/man/filelib.html#wildcard-1 for
                the wildcard patterns.
```

**Comment by lukebakken** [comment]
@NelsonVides I thought I'd bring this to your attention. Thanks!

**Comment by NelsonVides** [comment]
@lukebakken Ah, I see what happened! That reminds me, that the best thing this repo needs next is proper testing. I just didn't try wildcards :cry: 
I'm fixing this asap, hold on!

**Comment by NelsonVides** [comment]
@lukebakken solved on #36, just wait for merge ;)
